experimental/dds/tree2/src/core/tree/anchorSet.ts

```diff
@@ -857,6 +901,8 @@ export class AnchorSet implements ISubscribable<AnchorSetRootEvents>, AnchorLoca
 					// Remove subtree path visitors added at this node if there are any
 					this.pathVisitors.delete(p);
 				});
+				// QUESTION
+				// Can this.parent change because of the call above? Otherwise we're doing a redundant assert.
```

Comment:
Checking in case we can clean up a bit; haven't had a chance to fully understand what side effects can result from that call.

Reply:
That does look redudant to me. Feel tree to remove the lower one at part of this PR, otherwise a separate PR is fine too.

Reply:
Opened https://github.com/microsoft/FluidFramework/pull/17745